src/fsharp/Fable.FCS/app.fs

```diff
@@ -0,0 +1,69 @@
+module App
```

Comment:
How about changing ``Fable.FCS`` to ``fcsj``?  It is after all an F# compiler for Javascript.  Or ``fcs-js`` or ``fcs-fable``?  Just whatever is most idiomatic